ftva_lab_data/templates/partials/search_results_table.html

```diff
@@ -92,16 +109,33 @@
 </table>
 
 <div class="d-flex justify-content-center align-items-center gap-2">
-
-    <button class="btn btn-primary" {% if page_obj.has_previous %}
-        hx-get="{% url 'render_table' %}?page={{ page_obj.previous_page_number }}&search={{ search }}&search_column={{ search_column }}"
-        hx-target="#table-container" {% else %} disabled {% endif %}>Previous</button>
-
-    <span>Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}</span>
-
-    <button class="btn btn-primary" {% if page_obj.has_next %}
-        hx-get="{% url 'render_table' %}?page={{ page_obj.next_page_number }}&search={{ search }}&search_column={{ search_column }}"
-        hx-target="#table-container" {% else %} disabled {% endif %}>Next</button>
+    <button class="btn btn-primary"
```

Comment:
This page navigation code seems to be exactly the same here and above at lines 1-29.  Can it be moved to another partial template and included in both places, instead of explicit repetition?